aptos-move/framework/aptos-stdlib/sources/cryptography/multi_ed25519.move

```diff
@@ -191,47 +222,62 @@ module aptos_std::multi_ed25519 {
         message: vector<u8>
     ): bool;
 
+    #[test_only]
+    native fun generate_keys_internal(total: u8, threshold: u8): (vector<u8>,vector<u8>);
+
+    #[test_only]
+    native fun sign_internal(sk: vector<u8>, message: vector<u8>): vector<u8>;
+
     //
-    // Test cases
+    // Tests
     //
 
+    #[test_only]
+    struct TestMessage has copy, drop {
+        foo: vector<u8>,
+        bar: u64,
+    }
+
+    #[test_only]
+    fun pollute_first_signature(sig: &mut Signature) {
+        let first_sig_byte = vector::borrow_mut(&mut sig.bytes, 0);
+        *first_sig_byte = *first_sig_byte + 1;
```

Comment:
Will this overflow with an error when `first_sig_byte` is 255? I think so, according to [this](https://move-language.github.io/move/integers.html#arithmetic).

@vgao1996, can you confirm?

Reply:
To deal with this, I recommend doing a bitwise XOR (`^`) with `0xFF` which will flip all bits. See [here](https://move-language.github.io/move/integers.html#bitwise).